src/NuGet.Core/NuGet.Packaging/Signing/Utility/AttributeUtility.cs

```diff
@@ -234,11 +194,30 @@ private static List<AsnEncodedData> ToList(this AsnEncodedDataCollection collect
         private static List<SignatureType> GetCommitmentTypeIndicationRawValues(CryptographicAttributeObject attribute)
         {
             var values = new List<SignatureType>(1);
```

Comment:
Can you add a comment to why start the list with size 1? Maybe in the future this doesn't make sense and a comment would make it easier to notice in that case.